src/main/java/org/gridsuite/study/server/RemoteServicesProperties.java

```diff
@@ -4,33 +4,50 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package org.gridsuite.study.server.service;
+package org.gridsuite.study.server;
 
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.web.context.WebServerApplicationContext;
 import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
 
 import java.util.List;
 import java.util.Objects;
 
 /**
  * @author David Braquart <david.braquart at rte-france.com>
  */
+@Validated
 @Component
 @ConfigurationProperties(prefix = "gridsuite")
+@AllArgsConstructor
+@NoArgsConstructor
 @Data
 public class RemoteServicesProperties {
```

Comment:
Tbh I dont really understand why this class exist, shouldn't this code be in your new RemoteServices class instead ?

Reply:
No, in Spring we map configuration in a class to store and manage it.